tests/button_led/main.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2022 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Minimal test to connect every LED with a button state
+ *
+ * @author      Benjamin Valentin <benjamin.valentin@ml-pa.com>
+ *
+ * @}
+ */
+
+#include "board.h"
```

Comment:
Why add this test? there is already a `tests/leds` and `tests/buttons`.

Reply:
I don't this test is needed, better adapt the already present tests no?

Reply:
Ok, I added it to `tests/leds`

Reply:
Can you extended the README accordingly?|

Reply:
Sure!